samplesheets/vueapp/package-lock.json

```diff
@@ -38,6 +38,7 @@
         "eslint-plugin-vue": "^7.20.0",
         "fetch-mock-jest": "^1.5.1",
         "jest": "^27.1.0",
+        "node-fetch": "^3.3.2",
```

Comment:
This is not needed, and package-lock.json should not be modified for this pull request. Please revert to the original packages.